### ISSUE TITLE
[Terminal> Rename Window]: Keyboard users are not able to access 'Window 1' pop-up when they open it using defined shortcut key.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12021]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator
Tool: Accessibility Insights for Windows Version 1.1.1741.1

### Steps to reproduce

Repro Steps:
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Navigate to Settings> Action> Add new button.
4. Add shortcut 'Ctrl + Shift +r' for 'Rename Window' actions.
5. Save Changes using 'Save' button.
6. Then press 'Ctrl + Shift + r' to open 'Window 1' pop-up.
7. Try to access 'Window 1' pop-up using keyboard. 
 
**User Experience:**
As pop-up control are not accessible with Keyboard as a result, they will not able rename Window name.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

**Attachments:**
[Keyboard user is not able to access Window 1 Pop-up. (2).zip](https://github.com/microsoft/terminal/files/7769231/Keyboard.user.is.not.able.to.access.Window.1.Pop-up.2.zip)

### Expected Behavior

Keyboard users should directly jump to 'Window 1' edit field when user open 'Window 1' pop-up using defined shortcut key.

### Actual Behavior

Keyboard users are not able to access 'Window 1' pop-up when they open it using defined shortcut key.

**Note Issue:** Screen Reader users are not able to access 'Window 1' pop-up when they open it using defined shortcut key. Screen Reader is announcing 'Press F6 key to access...' but nothing happened on pressing F6.

**Comment by zadjii-msft** [comment]
We're already tracking this in #11322, which is blocked on the upstream https://github.com/microsoft/microsoft-ui-xaml/issues/1607 - there's no way for us to reliably focus a `TeachingTip` when it's opened.

**Comment by carlos-zamora** [comment]
@v-rpundir Just want to confirm. Sinze This is blocked by https://github.com/microsoft/microsoft-ui-xaml/issues/1607, you're marking this as a "tracking external" bug, correct?

**Comment by v-rpundir** [comment]
Yes @carlos-zamora, we are tracking this external for now.

**Comment by carlos-zamora** [comment]
@v-rpundir This should now be fixed in https://github.com/microsoft/terminal/releases/tag/v1.13.10983.0

Can you confirm and close the bug please?

**Comment by v-rpundir** [comment]
Closing as fixed now on Windows Terminal Preview Version: 1.13.10984.0
